### PR TITLE
Update chunk_extractor.py to convert text to json

### PR DESCRIPTION
**Pull request by rishiraj**
Processess as json if input type is application json. Also gives output as json.

### CODE REVIEW

File: text/chunking/chunk_extractor.py [hunk 1]
```diff
@@ -40,6 +46,8 @@ def extract(
                 )
                 if chunk.metadata:
                     chunk_content.features.append(Feature.metadata(chunk.metadata))
+            elif content.content_type == "application/json":
+                chunk_content = Content(content_type="application/json", data=json.dumps(chunk), features=content.features, labels=content.labels)
```
Comment (diptanu) [inline text/chunking/chunk_extractor.py:50]: Content.from_json

**Review by diptanu — CHANGES_REQUESTED** [review]

File: text/chunking/chunk_extractor.py (hunk 1 above)
Comment (rishiraj) [reply text/chunking/chunk_extractor.py]: resolved

File: text/chunking/chunk_extractor.py [hunk 2]
```diff
@@ -40,6 +46,12 @@ def extract(
                 )
                 if chunk.metadata:
                     chunk_content.features.append(Feature.metadata(chunk.metadata))
+            elif content.content_type == "application/json":
+                chunk_content = Content.from_json(
+                    json.dumps(chunk),
```
Comment (diptanu) [inline text/chunking/chunk_extractor.py:51]: You don't need to dump json here. It is being done internally.

**Review by diptanu — CHANGES_REQUESTED** [review]

File: text/chunking/chunk_extractor.py (hunk 2 above)
Comment (rishiraj) [reply text/chunking/chunk_extractor.py]: resolved

**Pull request merged**